Makefile

```diff
@@ -114,7 +114,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	docker build -t ${IMG} . --load
```

Comment:
No need for --load since buildx is not being used; `docker build` automatically stores the built image in the local Docker image store.

Reply:
i think podman uses buildx internally.. can't build with out if using podman desktop